core/platform/monitoring.go

```diff
@@ -0,0 +1,15 @@
+package platform
+
+// Observability keys
+const (
+	KeyCapabilityID        = "capabilityID"
```

Comment:
@jmank88 is parity with the http library that important here? The verbosity of these labels is now much higher for little gain imo

Reply:
What do you mean? I only suggested reordering. Prefix instead of suffix keeps the namespace clean by naturally grouping things when sorted, like in the package docs.